Assets/Scripts/EnemySystem/ScriptableObjects/EnemyAttackStrategies/GunnerStrategy.cs

```diff
@@ -21,8 +21,7 @@ public class GunnerStrategy : EnemyAttackStrategy
          */
         public override bool Attack(GameObject player, GameObject emu, int damageGiven, bool hasCollided)
         {
-            if ((Math.Abs(emu.transform.position.x) - Math.Abs(player.transform.position.x) < shootDistanceRangeMax)
-                && (Math.Abs(emu.transform.position.y) - Math.Abs(player.transform.position.y) < shootHeightRangeMax))
+            if (Vector2.Distance(player.transform.position, emu.transform.position) < shootDistanceRangeMax)
```

Comment:
I decided to change this file because it was causing an issue with the sound effects where the gunner emu would shoot (you could hear it) but the enemy was super far away

Reply:
oooooo i see, that's fine, then!